tests/test_autosynth_apiary_provider.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from unittest.mock import patch
+
+from autosynth.github import GitHub
+from autosynth.providers import apiary
+
+
```

Comment:
Could you also add negative tests: confirm some strings do not match the regex?

Reply:
Done. We relaxed the pattern here, so any .json with at least 2 segments matches. This seems fine for now as there's currently nothing else in the discoveries/ folder of discovery-artifact-manager.